include/envoy/config/grpc_mux.h

```diff
@@ -15,8 +15,8 @@ namespace Config {
 // clang-format off
 #define ALL_CONTROL_PLANE_STATS(COUNTER, GAUGE)                                                    \
   COUNTER(rate_limit_enforced)                                                                     \
-  GAUGE(connected_state)                                                                           \
-  GAUGE(pending_requests)                                                                          \
+  GAUGE(connected_state, NeverImport)                                           \
+  GAUGE(pending_requests, Accumulate)                                           \
```

Comment:
The trailing '\' on this line will actually prevent clang-format from turning back on. I hate to ask for this, but given the non-uniform slashes in this change, WDYT about just removing the clang-format off/on around these types of blocks throughout this change and just letting it be formatted? We have seen various issues in this area over the years and as much as this was originally about my formatting OCD I don't think it's worth it and we should just allow the formatter to do its stuff. Do you mind?

Reply:
I'm fine either way. For some reason my Emacs setup is not good at keeping these `\` aligned and I just forgot to clean it up in some cases.

I've now removed some of the clang-format keep-outs; let me see what the format-fixer makes this look like.

Reply:
Oh yeah. It's much easier to let clang-format fix the backslashes.

WDYT of grouping the counters together, then the gauges, then the histograms? Basically "M-x sort-lines on that region. I think that'll make things line up better in those groups for huge collections like ALL_HTTP_CONN_MAN_STATS and for smaller collections it doesn't matter that much :)

Reply:
one downside is that if anyone else merges any code that adds counter macro invocations while we finish this review, the merging will be messy.

Reply:
I'm fine with whatever order you want. This is a good opportunity to change this all up. I don't think the merge conflicts will be too bad so I would just go for it. Thank you!